examples/hello_world.rs

```diff
@@ -64,7 +64,7 @@ fn main() {
         "Markdown viewer",
         eframe::NativeOptions::default(),
         Box::new(move |cc| {
-            egui_extras::loaders::install(&cc.egui_ctx);
+            // egui_extras::loaders::install(&cc.egui_ctx);
```

Comment:
`egui_extras::loaders::install` is now immediate-mode safe, but should still be kept off of the hot-path, so the demo now works without this. `CommonMarkCache::new` calls `egui_extras::loaders::install`.